docs/docusaurus/docs/oss/changelog.md

```diff
@@ -14,6 +14,40 @@ When we deprecate our public APIs, we will
 
 Before we completely remove the functionality in a new major release, there will be at least one minor release that contains the deprecation so that you can smoothly transition to the new API.
 
+### 1.2.1
+* [BUGFIX] Add redirect for deploy-gx-agent ([#10573](https://github.com/great-expectations/great_expectations/pull/10573))
+* [BUGFIX] Fix typing on `mostly` and `value_set` fields ([#10571](https://github.com/great-expectations/great_expectations/pull/10571))
+* [BUGFIX] `ExpectationSuite` equality should ignore expectation ordering ([#10562](https://github.com/great-expectations/great_expectations/pull/10562))
+* [BUGFIX] Get `MicrosoftTeamsNotificationAction` working with V1 ([#10593](https://github.com/great-expectations/great_expectations/pull/10593))
+* [BUGFIX] Support string substitution for EmailAction ([#10600](https://github.com/great-expectations/great_expectations/pull/10600))
+* [BUGFIX] Ensure that data docs are accessible through MS Teams action results ([#10599](https://github.com/great-expectations/great_expectations/pull/10599))
+* [BUGFIX] Incorrect row count when `UnexpectedRowsExpectation` returns more than 200 rows ([#10604](https://github.com/great-expectations/great_expectations/pull/10604))
+* [DOCS] Edit API reference internal links to prevent 404s ([#10528](https://github.com/great-expectations/great_expectations/pull/10528))
+* [DOCS] data quality distribution analysis, working draft ([#10440](https://github.com/great-expectations/great_expectations/pull/10440)) (thanks @Quantisan)
```

Comment:
Could you remove the `(thanks @Quantisan)`. This is a devrel contractor and should be added to the teams.yml file (in a separate PR)?

Reply:
I'll add them in another PR. I did see a thanks @adeola-ak from a previous release, so removed that here while I was at it.